1-org/modules/centralized-logging/README.md

```diff
@@ -87,6 +89,7 @@ module "logging_logbucket" {
 |------|-------------|
 | bigquery\_destination\_name | The resource name for the destination BigQuery. |
```

Comment:
is bigquery_destination_name still needed after removing the bq sink?

Reply:
done